pages/index.js

```diff
@@ -37,6 +38,19 @@ const HeroUnit = styled.div`
   padding-top: 16px;
 `
 
+const ExploreButton = styled(Button)`
+  &:hover, &:focus {
+    background-color: white;
+    color: ${props => props.theme.colors.blue5};
+  }
+
+  &:active {
+    background-color: white;
+    color: ${props => props.theme.colors.blue5};
+    box-shadow: 0 0 0 0.2rem ${props => props.theme.colors.blue4};
```

Comment:
We should probably move this thing into the design-system as it's applicable to other components too.

xref: https://github.com/ooni/design-system/issues/53